pipeline/continuation/model.py

```diff
@@ -0,0 +1,123 @@
+"""
+Continue the training pipeline with an existing model.
```

Comment:
Hm, it's not immediately clear how this code plays together with the old training continuation code... It seems it will replace the train-* steps in the graph by this one with downloading the models. Do we need to keep the old transform that adds mounts then?

Reply:
I clarified how this works in the comment and filed a follow-up. #1151

```
Continue the training pipeline with an existing model. This produces the `continue-model`
task, which is dynamically used (at taskgraph generation time) as a dependency rather
than a `train`

TODO(#1151) - This only support backwards models with the "use" mode. We may be able
to unify everything here.
```